api/build.gradle.kts

```diff
@@ -32,6 +32,11 @@ dependencies {
     // DB
     implementation("org.litote.kmongo:kmongo:4.11.0")
 
+    // SE
+    implementation("org.apache.httpcomponents.core5:httpcore5:5.2.4")
+    implementation("org.apache.httpcomponents.core5:httpcore5-h2:5.2.4")
+    implementation("org.opensearch.client:opensearch-java:2.10.0")
```

Comment:
The `httpcore` libraries provide HTTP request support for the Opensearch client, which makes requests over HTTP.

There _are_ a range of Kotlin-specific Opensearch clients; I've not yet found one that wasn't a complete pig to set up or wasn't completely baffling to wrap my head around. I'm certain they exist, but didn't have any luck on a quick first pass.

Although the Java client is **extremely verbose** I've used it for this first version as it's very concise to form the initial connection to Opensearch, and constructing the query can be tidied up with helper methods if we want to go down that road.